.devops/deploy-pipelines.yml

```diff
@@ -206,3 +221,19 @@ stages:
                 sourceSlot: staging
                 swapWithProduction: true
               displayName: Swap with production slot
+
+
+  # Publish client SDKs to NPM
+  - stage: PublishClientSDKstoNPM
+    dependsOn: Release
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - ${{ each spec in parameters.OPENAPI_SPEC_PATHS }}:
+        - job: publish_SDK_${{ spec.Key }}   
+          steps:
+          # Template for generating and deploying client SDk to NPM
+          - template: templates/client-sdk-publish/template.yaml@pagopaCommons
+            parameters:
+              openapiSpecPath: '${{ spec.Value }}'
+              artifactName: 'Bundle_${{ spec.Key }}'
```

Comment:
We should specify the package name for each sdk

Reply:
> We should specify the package name for each sdk

Can we think about something like _'@pagopa/io-backend-**auth**-sdk'_

Reply:
We must find a name for each fo the following:
```yaml
api_auth: 'api_auth.yaml'
      api_backend: 'api_backend.yaml'
      api_bonus: 'api_bonus.yaml'
      api_bpd: 'api_bpd.yaml'
      api_myportal: 'api_myportal.yaml'
      api_notifications: 'api_notifications.yaml'
      api_pagopa: 'api_pagopa.yaml'
      api_public: 'api_public.yaml'
      api_session: 'api_session.yaml'
```

Reply:
I konw, I was moving to a list of objects, I'm gonna push a proposal and then we decide

Reply:
Done in [105db74](https://github.com/pagopa/io-backend/pull/772/commits/105db74a1972d682baf9470f8af578618eaf0cc0)